sdk/src/main/java/org/coralibre/android/sdk/internal/device_info/DeviceList.java

```diff
@@ -0,0 +1,81 @@
+package org.coralibre.android.sdk.internal.device_info;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import androidx.core.content.SharedPreferencesCompat;
+
+import org.coralibre.android.sdk.BuildConfig;
+import org.coralibre.android.sdk.R;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+
+import static org.coralibre.android.sdk.internal.device_info.ConfidenceLevel.NONE;
+import static org.coralibre.android.sdk.internal.device_info.ConfidenceLevel.getConfidenceLevel;
+
+public class DeviceList {
+    private static final String TAG = DeviceInfo.class.toString();
+
+    private static final String DEVICE_INFO_KEY = "DeviceList_device_info_key";
+
+    private static final int LIST_FILE_ID = R.raw.en_calibration_2020_08_12;
```

Comment:
Maybe add a comment here about the fact that this file probably needs to be updated as new devices are calibrated.